src/twister2/specification_processor.py

```diff
@@ -37,6 +37,7 @@ class SpecificationProcessor(abc.ABC):
 
     def __init__(self, twister_config: TwisterConfig) -> None:
         self.twister_config = twister_config
+        self.zephyr_base: str = self.twister_config.zephyr_base
```

Comment:
there is a problem at my side, whenever I want to use pytest, it always ask me to set the zephyr_base. even it is not a twister scripte, this occures to me like the twister plugin contaminate the pytest envrionment?

Reply:
This shouldn't be the case. We modify the plugin in a way that it has to be explicitly turned on, by adding `--twister` to pytest call. To mitigate exactly such issues as you describes. It should only ask you for zephyr base if you use `--twister` in pytest call. Maybe you have some older version of the plugin installed?

Reply:
there is an older version, I remove them then it is fine now.